components/account/AccountUserName.jsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from "react";
+import styles from "@/styles/components/account/account-user-name.module.scss";
+
+export default function AccountUserName() {
+	const [isEditActive, setIsEditActive] = useState(false);
+	const [userName, setUserName] = useState("Lio Giladi");
+	const [changingUserName, setChangingUsername] = useState(userName);
+
+	const onEditbuttonClick = () => {
+		setIsEditActive((prev) => {
+			if (prev === true) {
+				setUserName(changingUserName);
+			}
+
+			return !prev;
+		});
+	};
+
+	const onUserNameChange = (event) => {
```

Comment:
I know it's your style to declare a function for every event, but it _would_ be cleaner to just:
```jsx
<input otherProps onChange={(e) => setUserName(e.target.value)} />
```

Reply:
noted